pkg/file/path_queue.go

```diff
@@ -0,0 +1,43 @@
+package file
+
+type PathQueue struct {
```

Comment:
I couldn't find where this is getting used, is this something we need?

Reply:
It would have been used in the BFS, I implemented it out of habit. I'll remove.